Lib/test/test_cmd_line.py

```diff
@@ -1051,6 +1051,81 @@ def test_int_max_str_digits(self):
         )
         self.assertEqual(res2int(res), (6000, 6000))
 
+    def test_cmd_dedent(self):
+        # test that -c auto-dedents its arguments
+        from textwrap import dedent
+        test_cases = [
```

Comment:
We need a test case with empty strings as well and a line full of (possibly mixed) whitespaces.